utils/gen_python.jl

```diff
@@ -0,0 +1,57 @@
+using Configurations
+using Dates
+using InteractiveUtils
+using Legolas
+using OteraEngine
+using Ribasim
+
+pythontype(::Type{<:AbstractString}) = "Series[str]"
+pythontype(::Type{<:Integer}) = "Series[int]"
+pythontype(::Type{<:AbstractFloat}) = "Series[float]"
+pythontype(::Type{<:Number}) = "Series[float]"
+pythontype(::Type{<:Bool}) = "Series[pa.BOOL]" # pa.BOOL is a nullable boolean type, bool is not
```

Comment:
```suggestion
pythontype(::Type{<:Bool}) = "Series[pa.BOOL]" # pa.BOOL is a nullable boolean type, bool is not nullable
```

I assume that's what you mean? Had to read the sentence a few times to grasp it :)